sources/assets/shells/history.d/impacket

```diff
@@ -66,4 +66,8 @@ psexec.py -hashes :"$NT_HASH" "$DOMAIN"/"$USER"@"$TARGET"
 dcomexec.py -hashes :"$NT_HASH" "$DOMAIN"/"$USER"@"$TARGET"
 dcomexec.py -hashes :"$NT_HASH" "$DOMAIN"/"$USER"@"$TARGET"
 atexec.py -hashes :"$NT_HASH" "$DOMAIN"/"$USER"@"$TARGET"
-smbpasswd.py -newpass '123Pentest!!!' "$DOMAIN"/"$USER":"$PASSWORD"@"$DC_HOST"
+changepasswd.py -newpass '123Pentest!!!' "$DOMAIN"/"$USER":"$PASSWORD"@"$DC_HOST"
+owneredit.py -action write -new-owner "$NEW_OWNER" -target "$TARGET_OBJECT" "$DOMAIN"/"$USERNAME":"$PASSWORD"
+mssqlclient.py "$DOMAIN"/"$USER":"$PASSWORD"@"$TARGET"
+describeTicket.py "TGT.ccache"
+changepasswd.py -hashes :"$NT_HASH" "$DOMAIN"/"$USER"@"$TARGET" -newpass '123Pentest!!!'
```

Comment:
Use this instead to have augument in the same order as the other (easier to read and compare with ctrl+R)
`changepasswd.py -newpass '123Pentest!!!' -hashes :"$NT_HASH" "$DOMAIN"/"$USER"@"$TARGET"`